ml-agents/tests/yamato/check_coverage_percent.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import print_function
+import sys
+import os
+
+SUMMARY_XML_FILENAME = "Summary.xml"
+
+
+def check_coverage(root_dir, min_percentage):
+    summary_xml = None
+    for dirpath, _, filenames in os.walk(root_dir):
+        if SUMMARY_XML_FILENAME in filenames:
+            summary_xml = os.path.join(dirpath, SUMMARY_XML_FILENAME)
+            break
+    if not summary_xml:
+        print("Couldn't find Summary.xml in root directory")
+        sys.exit(0)
```

Comment:
I am not aware. What generates "Summary.xml"? In what circumstances is it not generated? Should this be an error code 1 if the file was not present?

Reply:
It's an artifact of code coverage (which should always be in CI as of https://github.com/Unity-Technologies/ml-agents/pull/3633). 

Good point, that was a typo and it should be exit(1)